assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -121,12 +129,24 @@ avc_cache_threshold=8192
 initrd_remove_dir=
 initrd_dst_img=
 initrd_add_dir=
+
 # overrides cpu-partitioning cmdline
-cmdline_cpu_part=+nohz=on rcu_nocbs=${isolated_cores} tuned.non_isolcpus=${not_isolated_cpumask} intel_pstate=disable nosoftlockup
+cmdline_cpu_part=+nohz=on rcu_nocbs=${isolated_cores} tuned.non_isolcpus=${not_isolated_cpumask} systemd.cpu_affinity=${not_isolated_cores_expanded} intel_iommu=on iommu=pt
+
 {{if .StaticIsolation}}
-cmdline_realtime=+tsc=nowatchdog intel_iommu=on iommu=pt isolcpus=domain,managed_irq,${isolated_cores} systemd.cpu_affinity=${not_isolated_cores_expanded}
+cmdline_isolation=+isolcpus=domain,managed_irq,${isolated_cores}
```

Comment:
the only other thing I'm not comfortable with in this PR is that in this commit we are both changing the profiles proper and the way we create them. I would be happier if, if possible, we can have 1. a commit which takes the current profiles unchanged and just changes the way we combine the existing data, using the workload hints. 2. a commit which changes the profiles per se, but NOT the way we combine the data. IOW, I'd love to split the responsabilities in this commit.
This is NOT BLOCKING, depending on what the other reviewers prefer.

Reply:
done